### PR TITLE
chore: Add JSDoc type "ConversationData"

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: app/script/conversation/ConversationMapper.js [hunk 1]
```diff
@@ -22,21 +22,53 @@
...
- * @typedef {object} SelfStatusUpdate
- * @property {number=} archived_timestamp
- * @property {number=} cleared_timestamp
- * @property {number=} ephemeral_timer
- * @property {number=} message_timer
- * @property {number=} last_event_timestamp
- * @property {number=} last_read_timestamp
- * @property {number=} last_server_timestamp
- * @property {boolean=} otr_archived
+ * @typedef {object} ConversationData
```
Comment (atomrc) [inline app/script/conversation/ConversationMapper.js]: Shouldn't we precise if it is data from the backend or the db?

File: app/script/conversation/ConversationMapper.js [hunk 2]
```diff
@@ -22,21 +22,53 @@
...
+ */
+
+/**
+ * @typedef {object} OtherMember
+ * @property {string} id
+ * @property {number} status
+ */
+
+/**
+ * @typedef {object} SelfStatusUpdate
```
Comment (atomrc) [inline app/script/conversation/ConversationMapper.js]: This one seems to be incomplete, I believe we can pass a few more properties to be updated in the self status

Comment (bennycode) [reply app/script/conversation/ConversationMapper.js]: Good catch! Fixed.

File: app/script/conversation/ConversationMapper.js (hunk 1 above)
Comment (bennycode) [reply app/script/conversation/ConversationMapper.js]: #changed

**Pull request merged**